scripts/run-e2e-config.ts

```diff
@@ -159,8 +159,6 @@ export const sfcVue: Parameters = {
   generator: fromDeps('vue', 'vue-loader', 'vue-template-compiler'),
   additionalDeps: [
     'webpack@webpack-4',
-    // TODO: remove when https://github.com/storybookjs/storybook/issues/11255 is solved
-    'core-js',
```

Comment:
I meant to remove all mentions of `core-js` from this file, with the change in this PR it Should Work™